assets/src/edit-story/components/library/panes/text/textSets/raw/list_1.json

```diff
@@ -0,0 +1,5503 @@
+{
```

Comment:
^ This as well, we could rename to just `list.json`.

Reply:
Cool, I'm just gonna add the list 2 sets to this PR too then